codegen/pluralization.ts

```diff
@@ -1,5 +1,6 @@
 import { pascalCase, deconstruct } from './text-manipulation';
-
+import { linq } from '@azure-tools/linq';
+linq;
```

Comment:
Probably not needed?

```suggestion
```

Reply:
It needs a reference to linq for it to pick up the Array extensions.